src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -16,7 +18,17 @@ import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribut
     '[attr.aria-busy]': 'true',
   },
 })
-export class ClrSpinner {
+export class ClrSpinner implements AfterViewInit {
+  constructor(private el: ElementRef, private ariaLiveService: AriaLiveService) {}
+
+  ngAfterViewInit() {
+    const textNode = this.el.nativeElement.childNodes[0];
+    if (textNode) {
+      // Tested: when the string is different, Chrome is reading it
+      this.ariaLiveService.announce(textNode.nodeValue, this.setAriaLive);
+    }
+  }
```

Comment:
This will need a browser platform check `if (isPlatformBrowser(this.platformId)) {}`